ibis/expr/datatypes.py

```diff
@@ -1637,9 +1720,18 @@ def can_cast_variadic(
 
 # geo spatial data type
 # cast between same type, used to cast from/to geometry and geography
-@castable.register(Array, (Point, LineString, Polygon, MultiPolygon))
-@castable.register((Point, LineString, Polygon, MultiPolygon), Geometry)
-@castable.register((Point, LineString, Polygon, MultiPolygon), Geography)
+@castable.register(
```

Comment:
would be good to create a new variable to group all geo types? 
maybe something like:

```
GEO_TYPES = (Point, LineString, Polygon, MultiLineString, MultiPoint, MultiPolygon)

@castable.register(Array, GEO_TYPES)
@castable.register(GEO_TYPES, Geometry)
@castable.register(GEO_TYPES, Geography)
def can_cast_geospatial(source, target, **kwargs):
    return True

Reply:
Done!

Reply:
great! thanks!